src/platform/ESP32/ConnectivityManagerImpl_WiFi.cpp

```diff
@@ -350,6 +350,8 @@ CHIP_ERROR ConnectivityManagerImpl::_GetAndLogWiFiStatsCounters(void)
     uint16_t freq;
     uint16_t bssid;
 
+    IgnoreUnusedVariable(freq);
+    IgnoreUnusedVariable(bssid);
```

Comment:
This change does not fit into the PR description. Does this fix warnings?

Reply:
Yes, to defeat unused variable warnnings when disable CHIP_ERROR